.github/workflows/push-foundation-repo.yml

```diff
@@ -0,0 +1,17 @@
+name: Sync Cardinal-Cryptography repo with Aleph-Zero-Foundation repo
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'Cardinal-Cryptography/aleph-node'}}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.SYNCAZF }}
```

Comment:
There is the same token for both AlephBFT and aleph-node?

Reply:
Yes. More specifically this token has write rights to aleph-foundation organization repos.

Reply:
I'm not sure if it is safe to have one token for all organization repos, but won't also fight for changing it

Reply:
I don't think there is a problem here. We keep all our work in Cardinal Cryptography organization, so having write token for repos in Aleph Foundation organization won't mess with our work in any way